src/main/java/net/sacredlabyrinth/phaed/simpleclans/chat/handlers/SpigotChatHandler.java

```diff
@@ -47,6 +47,24 @@ public void run() {
         }.runTask(plugin);
     }
 
+    private String stripColorsAndFormatsPerPermission(ClanPlayer sender, String message) {
+        if (!plugin.getPermissionsManager().has(sender.toPlayer(), "simpleclans.member.chat.color")) {
```

Comment:
PermissionManager can be added to ChatHandler interface since it's a [second usage](https://github.com/RoinujNosde/SimpleClans/blob/master/src/main/java/net/sacredlabyrinth/phaed/simpleclans/chat/handlers/SpyChatHandler.java#L48).

Reply:
Actually, if you will move the method to ChatUtils it wouldn't need to update ChatHandler.